src/Modules/OrchardCore.Commerce.Payment.Stripe/Views/CheckoutStripe.cshtml

```diff
@@ -12,6 +12,7 @@
     string publishableKey = Model.PaymentProviderData?.PublishableKey ?? string.Empty;
     string clientSecret = Model.PaymentProviderData?.ClientSecret ?? string.Empty;
     string paymentIntentId = Model.PaymentProviderData?.PaymentIntentId ?? string.Empty;
+    string accountId = Model.PaymentProviderData?.AccountId ?? null;
```

Comment:
What does `?? null` do? I think that means "if null, make it null" which is pointless.

Also I suggest trimming this value like `Model.PaymentProviderData?.AccountId?.Trim()` and the using `string.IsNullOrEmpty` instead of `string.IsNullOrWhiteSpace` on line 49.

Reply:
Not sure if this is going to be faster just to save a IsNullOrWhiteSpace while that's what it is for. Though, I don't mind doing the change.

Reply:
done